sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/Generated/Models/HybridComputeMachinePatch.cs

```diff
@@ -19,7 +19,23 @@ public HybridComputeMachinePatch()
 
         /// <summary> Identity for the resource. Current supported identity types: SystemAssigned. </summary>
         public ManagedServiceIdentity Identity { get; set; }
-        /// <summary> Hybrid Compute Machine properties. </summary>
-        public MachineUpdateProperties Properties { get; set; }
+        /// <summary> Metadata pertaining to the geographic location of the resource. </summary>
+        public LocationData LocationData { get; set; }
+        /// <summary> Specifies the operating system settings for the hybrid machine. </summary>
+        public OSProfile OSProfile { get; set; }
+        /// <summary> The metadata of the cloud environment (Azure/GCP/AWS/OCI...). </summary>
+        internal CloudMetadata CloudMetadata { get; set; }
+        /// <summary> Specifies the cloud provider (Azure/AWS/GCP...). </summary>
+        public string CloudMetadataProvider
+        {
+            get => CloudMetadata is null ? default : CloudMetadata.Provider;
+        }
+
+        /// <summary> The info of the machine w.r.t Agent Upgrade. </summary>
+        public AgentUpgrade AgentUpgrade { get; set; }
+        /// <summary> The resource id of the parent cluster (Azure HCI) this machine is assigned to, if any. </summary>
+        public string ParentClusterResourceId { get; set; }
```

Comment:
Same question regarding the format of the ID

Reply:
changed the type of ParentClusterResourceId to Guid.

Reply:
fixed in the latest commit